pulpcore/pulpcore/app/viewsets/repository.py

```diff
@@ -128,6 +129,86 @@ class Meta:
         fields = ContentAdaptorFilter.Meta.fields
 
 
+class RepositoryVersionContentInFilter(CharInFilter):
+    """
+    Filter used to get the repository versions where some given content can be found.
+
+    Given a list of content_id, this filter will:
+        1. Get the RepositoryContent that the id can be found in
+        2. Get a list of version_added and version_removed where the content was
+           changed on the repository
+        3. Calculate and return the versions that the contents can be found on
+
+    """
+
+    def filter(self, qs, value):
+        """
+        Args:
+            qs (pulpcore.app.viewsets.repository.RepositoryVersionViewSet.queryset):
```

Comment:
This type is very informative, but I think it violates the spirit of the docstring conventions. It probably ought to be:

qs (django.db.models.query.QuerySet)